istioctl/cmd/istioctl/kubeinject.go

```diff
@@ -77,6 +78,29 @@ func getMeshConfigFromConfigMap(kubeconfig string) (*meshconfig.MeshConfig, erro
 	return cfg, err
 }
 
+// grabs the raw values from the ConfigMap. These are encoded as JSON.
+func getValuesFromConfigMap(kubeconfig string) (string, error) {
```

Comment:
I'm not sure about this part - didn't we have code to get the template for kube, that was also creating an interface ?
Can we extend or reuse that code ?